pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs

```diff
@@ -243,168 +151,219 @@
-                                       <form name=&quot;regrade-form&quot; method=&quot;POST&quot;>
-                                         <input type=&quot;hidden&quot; name=&quot;__action&quot; value=&quot;regrade&quot;>
-                                         <input type=&quot;hidden&quot; name=&quot;__csrf_token&quot; value=&quot;<%= __csrf_token %>&quot;>
-                                         <input type=&quot;hidden&quot; name=&quot;assessment_instance_id&quot; value=&quot;<%= row.assessment_instance_id %>&quot;>
-                                         <button type=&quot;button&quot; class=&quot;btn btn-secondary&quot; onclick=&quot;$('#row<%= iRow %>PopoverRegrade').popover('hide')&quot;>
-                                           Cancel
-                                         </button>
-                                         <button type=&quot;submit&quot; class=&quot;btn btn-primary&quot;>Regrade</button>
-                                       </form>
-                                       ">
-                    </div>
-
-                    <%# Capture all clicks to dropdown items to prevent scrolling to the top of the page %>
-                    <div class="dropdown-menu" onclick="window.event.preventDefault();">
-
-                      <% if (authz_data.has_instructor_edit) { %>
-                      <a class="dropdown-item"
-                         role="button" tabindex="0"
-                         data-toggle="modal" data-target="#deleteAssessmentInstanceModal"
-                         data-uid="<%= row.uid %>" data-name="<%= row.name %>"
-                         data-number="<%= row.number %>"
-                         data-date-formatted="<%= row.date_formatted %>"
-                         data-score-perc="<%= Math.floor(row.score_perc) %>"
-                         data-assessment-instance-id="<%= row.assessment_instance_id %>"
-                         >
-                        <i class="fa fa-times" aria-hidden="true"></i> Delete
-                      </a>
-                      <% } %>
-
-                      <% if (authz_data.has_instructor_edit) { %>
-                      <a role="button" tabindex="0" class="dropdown-item <% if (!row.open) { %>disabled<% } %>" <% if (row.open) { %>onclick="$('#row<%= iRow %>PopoverClose').popover('show')"<% } %>>
-                        <i class="fa fa-ban" aria-hidden="true"></i> Grade & Close
-                      </a>
-                      <% } %>
+        <table id="usersTable"
+               data-unique-id="assessment_instance_id"
+               data-classes="table table-sm table-hover table-bordered"
+               data-url="<%= urlPrefix %>/assessment/<%= assessment.id %>/instances/raw_data.json"
+               data-search="true"
+               data-show-columns="true"
+               data-show-refresh="true"
+               data-auto-refresh="true"
+               data-auto-refresh-status="false"
+               data-auto-refresh-interval="30"
```

Comment:
Is there any way to indicate how long until the next auto-refresh? Is this what the `data-auto-refresh-silent` option does?

Reply:
I don't know if it's possible to show that, I'll check. As for the silent option, it's to refresh it without redrawing the entire table, i.e., only update the rows/cells that changed.

Reply:
I can't find a easy and reliable way to show the time for the next auto-refresh through the BT extension, other than to bypass the extension and do the whole auto-refresh process separately, or to extend the BootstrapTable class functionality (which may not be reliable if the npm package is later updated and the changes become inconsistent). I don't think it's worth the effort.

Reply:
Agreed. It's fine as is.